documentation/latest/gsg/minikube.md

```diff
@@ -0,0 +1,147 @@
+---
+layout: documentation
+title: Run flotta on minikube
+---
+
+Flotta can run on top of any Kubernetes distributions, for testing and
+development you can follow this guide to get started running flotta on your
+devices.
+
+Let's start with minikube
+
+```shell
+---> minikube start
+😄  minikube v1.22.0 on Fedora 35
```

Comment:
I would comment that this is example output. We do not want people to rely on v1.22 especially when v1.25.2 is available.